packages/query/src/graphql/plugins/GetMetadataPlugin.ts

```diff
@@ -73,22 +73,27 @@ async function fetchFromTable(pgClient: any, schemaName: string): Promise<MetaDa
 
   metadata.queryNodeVersion = packageVersion;
 
-  const tableEstimates: [TableEstimate] = await pgClient
+  const tableEstimates = await pgClient
     .query(
-      `select relname, reltuples::bigint as estimate from pg_class where relname in (select table_name from information_schema.tables where table_schema = '${schemaName}')`
+      `select relname as table , reltuples::bigint as estimate from pg_class
+      where relnamespace in
+            (select oid from pg_namespace where nspname = $1)
+      and relname in
+          (select table_name from information_schema.tables
+           where table_schema = $1)`,
+      [schemaName]
     )
-    .then(({rows}) => rows.map(({estimate, relname}) => ({table: relname, estimate: estimate})))
     .catch((e) => {
       throw new Error(`Unable to estimate table row count: ${e}`);
     });
 
-  metadata.rowCountEstimate = tableEstimates;
+  metadata.rowCountEstimate = tableEstimates.rows;
 
   return metadata;
 }
 
 export const GetMetadataPlugin = makeExtendSchemaPlugin((build, options) => {
-  const schemaName = options.pgSchemas;
+  const [schemaName] = options.pgSchemas;
```

Comment:
There wasn't any error before?

Reply:
Nope. It gets cast to `string` fine but it was really `[string]`. Can't trust the types with a whole lot of `any` being used here, but the plugin needs it so no avoiding.